packages/cli/src/commands/connector/utils.ts

```diff
@@ -53,6 +53,10 @@ const validatePath = async (value: string) => {
 
 export const inquireInstancePath = async (initialPath?: string) => {
   const inquire = async () => {
+    if (!initialPath && (await validatePath('.')) === true) {
```

Comment:
Hmmm, why do we need to validate `'.'`?

Reply:
because it represents the current working directory?

Reply:
Yeah, but why not validating the `initialPath`? And also why `=== true`?